### DISCUSSION TITLE
How to import GIF

### DISCUSSION THREAD

**Discussion by Vince-vegas** [vercel/next.js#17995] [Help] [answered]
Why I'm getting this error after importing a **GIF** file into image **src**

```
import React from 'react';
import Spin from '../../public/images/spin.gif';

const Spinner = () => {
  return (
    <div className="spinner">
      <img src={Spin} alt="spin" />
    </div>
  );
};

export default Spinner;
```

**THE ERROR**
```
./public/images/spin.gif 1:7
Module parse failed: Unexpected character '' (1:7)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)
```

HOW TO IMPORT THE GIF File

**Comment by jamesmosier** [comment, marked as answer]
Hi @Vince-vegas. You can use the [next-images](https://www.npmjs.com/package/next-images) plugin in your `next.config.js` in order to import and display images!

Also there is an RFC for an [Image Component](https://github.com/vercel/next.js/discussions/16832) that should be coming in the near future from the looks of the RFC!

**Reply by Vince-vegas** [reply]
I just implement it and it works, thanks

**Reply by alex-r89** [reply]
Hi,

Sorry to revive this but how do you get this working with the new `next/image` ? Gifs dont seem to be supported.

**Reply by Vince-vegas** [reply]
**1st:** npm install next-images

**2nd:** create a config file like **next.config.js**

**3rd:** import this
```
const withImages = require('next-images')
module.exports = withImages()
```

**Reply by mhdalmajid** [reply]
dose it still the same problem or did they add a feature to nexjs to solve this issue?

**Comment by NyawiraMuturi** [comment]
next/image still doesn't support gifs. You have to use the native HTML <img/> tag. However, lazy loading and suspense help with optimization.
